tests/postgres_tests/test_search.py

```diff
@@ -346,6 +347,22 @@ def test_ranking_chaining(self):
         self.assertSequenceEqual(searched, [self.verse0])
 
 
+class SearchVectorIndexTests(PostgreSQLTestCase):
+    def test_search_vector_index(self):
+        """SearchVector generates IMMUTABLE SQL in order to be indexable."""
+        # This test should be moved to test_indexes and use a functional
+        # index instead once support lands (#26167)
+        query = Line.objects.all().query
+        resolved = SearchVector('id', 'dialogue', config='english').resolve_expression(query)
+        compiler = query.get_compiler(connection.alias)
+        sql, params = resolved.as_sql(compiler, connection)
+        # The following query will crash if the SQL is not IMMUTABLE.
+        with connection.cursor() as cursor:
+            cursor.execute(
+                'CREATE INDEX search_vector_index ON %s USING GIN (%s)' % (Line._meta.db_table, sql), params
```

Comment:
I presume this is cleaned up?

Reply:
It is. `PostgreSQLTestCase` subclasses `TestCase` which wraps every test in transactions and PostgreSQL supports transactional DDL.

Reply:
Pushed a commit that swaps it to `TextField`.

Reply:
Cool. I forget every time... 🥴